src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -576,6 +576,21 @@
     )
     private Set<ClaimsCostFrom> divorceClaimFrom;
 
+    @CCD(
+        label = "Uploaded supporting documents",
+        hint = "Such as marriage/civil union certificate, proof of name changes, certified translations etc.",
+        typeOverride = TextArea,
+        access = {DefaultAccess.class}
+    )
+    private String uploadedDocuments;
+
+
+    @CCD(
+        label = "Cannot upload supporting documents",
+        access = {DefaultAccess.class}
+    )
+    private Set<SupportingDocumentType> cannotUploadDocuments;
```

Comment:
this seems like it should be a boolean or text to inform why they cant.

Reply:
This will be list of documents that they can't upload:

<img width="542" alt="Screenshot 2021-04-23 at 15 18 17" src="https://user-images.githubusercontent.com/144435/115884752-26788d00-a447-11eb-9b6e-5a380a6596bd.png">